bandersnatch/bandersnatch.go

```diff
@@ -5,20 +5,24 @@ import (
 	"io"
 
 	gnarkbandersnatch "github.com/consensys/gnark-crypto/ecc/bls12-381/bandersnatch"
+	gnarkfr "github.com/consensys/gnark-crypto/ecc/bls12-381/fr"
```

Comment:
nit: note that although this is gnarkfr (bls12-381), its fp for bandersnatch, so maybe gnarkfp or something along those lines would be less confusing since we use fp already in this file